examples/load-people/vaccine.rs

```diff
@@ -14,18 +14,14 @@ pub enum VaccineTypeValue {
 }
 define_person_property!(VaccineType, VaccineTypeValue);
 define_person_property!(VaccineEfficacy, f64);
-define_person_property!(
-    VaccineDoses,
-    u8,
-    fn initialize(context: &Context, person_id: PersonId) {
-        let age = context.get_person_property(person_id, Age);
-        if age > 10 {
-            context.sample_range(VaccineRng, 0..5)
-        } else {
-            0
-        }
+define_person_property!(VaccineDoses, u8, |context: &Context, person_id| {
```

Comment:
Do you need to provide &Context here? I thought it could infer it.

Reply:
For some reason it can't seem to infer it, although I'm not sure why:
```
error[E0282]: type annotations needed
  --> examples/load-people/vaccine.rs:17:44
   |
17 | define_person_property!(VaccineDoses, u8, |context, person_id| {
   |                                            ^^^^^^^
18 |     let age = context.get_person_property(person_id, Age);
   |               ------- type must be known at this point
   |
```